h2o-r/h2o-package/R/models.R

```diff
@@ -3043,11 +3043,15 @@ h2o.cross_validation_predictions <- function(object) {
 #' @param data An H2OFrame object used for scoring and constructing the plot.
 #' @param cols Feature(s) for which partial dependence will be calculated.
 #' @param destination_key An key reference to the created partial dependence tables in H2O.
-#' @param nbins Number of bins used. For categorical columns make sure the number of bins exceed the level count.  If you enable add_missing_NA, the returned length will be nbin+1.
+#' @param nbins Number of bins used. For categorical columns make sure the number of bins exceed the level count.
```

Comment:
Existing grammar issue. Change "exceed" to "exceeds" in this.
"... the number of bins exceeds the level count."

Reply:
Done